WcaOnRails/app/controllers/api/v0/user_roles_controller.rb

```diff
@@ -81,7 +81,7 @@ class Api::V0::UserRolesController < Api::V0::ApiController
 
   # Returns a list of roles by user which are not yet migrated to the new system.
   private def group_roles_not_yet_in_new_system(group_id)
-    group = UserGroup.find(group_id)
+    group = UserGroup.find_by_id(group_id)
```

Comment:
`find_by_id` and `find` should be the same, as long as `id` is the primary key (which it should be in 99% of cases).
So why are you changing this?

Reply:
I actually faced an issue like find not there something. I don't remember the error exactly and don't know where it caused. So replaced `find` with `find_by_id`. Now I changed back and not getting the issue. Will check in deep if I face it again.